src/authenticator.rs

```diff
@@ -395,10 +395,8 @@ where
         reply.push((credential.label.len() + 1) as u8)?;
         reply.push(oath::combine(credential.kind, credential.algorithm))?;
         reply.extend_from_slice(&credential.label).map_err(|_| 0)?;
-        #[cfg(feature = "devel-ctaphid-bug")]
-        if reply.len() > 3072 {
-            // Finish early due to the usbd-ctaphid bug, which panics on bigger buffers than this
-            // FIXME Remove once fixed
+        if reply.len() > CTAPHID_MESSAGE_SIZE_LIMIT {
```

Comment:
Does CTAPHID have no chaining mechanism? 

If not should we have a paging mechanism for the `list` command. (Out of scope for this PR).

Reply:
The default USB packet size are 64 bytes and CTAPHID can split a message into multiple packets.  But `usbd-ctaphid` has a message buffer that is smaller than `ctaphid-dispatch`’s message buffer.

Reply:
Exactly. Can I somehow painlessly gather the underlying maximum common buffer size? Or, maybe I should not care?

Reply:
Not really, `ctaphid-dispatch` does not provide a constant for its message buffer size.